docs/_pages/0980-release-management.md

```diff
@@ -138,6 +138,8 @@ update your environment.
 
 Check the `make help` command first, as it includes important information.
 
+You can also check `make h` command that included release-related commands.
```

Comment:
`make h` is more “release-specific”, whereas `make help` is more general-purpose.